src/status_im/ui/screens/chat/views.cljs

```diff
@@ -286,6 +329,12 @@
     (utils/set-timeout #(re-frame/dispatch [:chat.ui/load-more-messages-for-current-chat])
                        (if platform/low-device? 700 200))))
 
+(defn pin-list-on-end-reached []
+  (if @state/scrolling
```

Comment:
we have this optimization for messages list , but im not sure we need it for pinned messages list, could we simplify it and just use regular list ?

Reply:
Indeed I think we don't need this code anymore as specifications changed to have a limit of 3 messages, so I'll simplify it